spectractor/extractor/extractor.py

```diff
@@ -693,12 +707,17 @@ def adjust_spectrogram_position_parameters(self):
         epsilon[epsilon == 0] = 1e-4
         fixed_default = np.copy(self.fixed)
         self.fixed = [True] * len(self.p)
+        # if fixed_default[3] is False and fixed_default[4] is False:
         self.fixed[3:5] = [False, False]  # shift_y and angle
+        # else:
+        #     for k, par in enumerate(self.input_labels):
+        #         if "y_c" in par and "_2" not in par:
+        #             self.fixed[k] = False
         self.sparse_indices = None
-        run_minimisation(self, "newton", epsilon, self.fixed, xtol=1e-4, ftol=100 / self.data.size)
+        run_minimisation(self, "newton", epsilon, self.fixed, xtol=1e-2, ftol=0.01)  # 1000 / self.data.size)
         self.fixed = fixed_default
         self.sparse_indices = None
-        self.set_mask(params=self.p)
+        self.set_mask(params=self.p, fwhmx_clip=3*parameters.PSF_FWHM_CLIP, fwhmy_clip=parameters.PSF_FWHM_CLIP)
```

Comment:
If you are going to have expressions in default args then reasserting them downstream isn't usually a good pattern. Happy to have a chat about all of this when we next speak though, so it's probably fine for now.